templates/data/opensearch/support.html

```diff
@@ -0,0 +1,544 @@
+{% extends 'base_index.html' %}
+
+{% block title %}
+  Get support for OpenSearch, an open source search engine, analytics tool and vector database.
+{% endblock %}
+
+{% block meta_description %}
+  Get support for OpenSearch, from Canonical. 24/7 or weekday support and up to ten years security patching per release
+  track.
+{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1ovtXv8RBybw-flvttyqWOzZivUryQMP34T1UGe90HlY/edit
+{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-strip is-shallow u-no-padding--bottom">
+    <div class="p-section">
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <div class="u-hide--small u-vertically-center u-aspect-ratio--16-9"
+               style="margin-top: -55px">
+            {{ image(url="https://assets.ubuntu.com/v1/6deb3921-opensearch-logo.png",
+                        alt="Opensearch",
+                        width="198",
+                        height="186",
+                        hi_def=True,
+                        loading="lazy") | safe
+            }}
+          </div>
+        </div>
+        <div class="col-9 col-medium-4">
+          <div class="p-section--shallow">
+            <h1>OpenSearch support</h1>
+            <p>
+              We are experts in OpenSearch. From providing everyday support to offering hands-on assistance
+              for complex projects, our experienced database performance specialists are prepared to assist
+              you with any challenge.
+            </p>
+            <p>
+              Deal with issues faster and guarantee the ongoing performance of your deployments with 24/7/365
+              consultative OpenSearch support.
+            </p>
+          </div>
+          <hr />
+          <a class="p-button--positive"
+             href="/contact-us"
+             aria-controls="data-opensearch-modal">Contact us</a>
```

Comment:
This button is not working for me on the demo, please check

Reply:
Yeah I know, it is because of a change in feature branch to which this code will merge. It would work after that